tensorflow/tools/ci_build/windows/cpu/cmake/run_py.bat

```diff
@@ -30,8 +30,14 @@ SET PIP_EXE="C:\Program Files\Anaconda3\Scripts\pip.exe"
 :: Uninstall tensorflow pip package, which might be a leftover from old runs.
 %PIP_EXE% uninstall -y tensorflow
 
+
+:: Since there are no wildcards in windows command prompt, use dark magic to get the wheel file name.
+DIR %REPO_ROOT%\%BUILD_DIR%\tf_python\dist\ /S /B > wheel_filename_file
+set WHEEL_FILENAME= <wheel_filename_file
```

Comment:
I think this might need to be `set /p WHEEL_FILENAME=<wheel_filename_file`. The `/p` means it reads a line of input (the "p" stands for "prompt"!) into the named variable.